cfg_VHcc_WLNu.py

```diff
@@ -117,6 +118,7 @@
     },
 
     workflow = VHccBaseProcessor,
+    workflow_options = {"dump_columns_as_arrays_per_chunk": "./Saved_columnar_arrays_WLNu"},
```

Comment:
Is it possible to dump arrays into the output directory of the run (the one we use in `-o` option)? It would be more convenient.